README.md

```diff
@@ -51,5 +50,42 @@ cat my_awesome_file.txt
 Command | Explanation
 --- | ---
 `cross-cat <filename>` | For example, in case of PowerShell, ensure that this module is run instead of `cat` built-in alias to `Get-Content`
-`npx cross-cat <filename>` | To run without installation, although npm information would be displayed as well
-`cat --help` | To get all the available options
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[table-cell-padding] Cell should be compact](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5222203)